packages/config/src/chainConfig/networks/ephemery.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import {fromHexString as b} from "@chainsafe/ssz";
+import {ChainConfig} from "../types.js";
+import {chainConfig as mainnet} from "../presets/mainnet.js";
+
+/* eslint-disable max-len */
+
+// Ephemery dynamic beacon chain config:
+// https://github.com/taxmeifyoucan/ephemeral-testnet/blob/master/specs.md
+
+// Iteration 0, "base"-genesis
+const ephemeryBaseChainConfig: ChainConfig = {
```

Comment:
as this is not exported, could make the code more readable below by omitting ephemery here

```suggestion
const baseChainConfig: ChainConfig = {
```